apidef/oas/server.go

```diff
@@ -94,6 +99,15 @@ func (s *Server) Fill(api apidef.APIDefinition) {
 	if ShouldOmit(s.EventHandlers) {
 		s.EventHandlers = nil
 	}
+
+	if s.IPAccessControl == nil {
```

Comment:
I wonder if we should extract those recurring code blocks into proper functions to better group them and make it easier to read

Reply:
these had to come in because of some edge cases. we need to make the flow continue and also make the OAS definition minimal (instead of having residual zero configs like in classic API definitions).
We can think of a better way to handle this if possible.